app/assets/javascripts/sitewide/atwho_res.js.erb

```diff
@@ -103,6 +103,11 @@ var SmartAnnotation = (function() {
 
             }
             $.getJSON(filterType.dataUrl, params, function(data) {
+
+              localStorage.setItem('scinote-smart-annotation-state-per-team-' + data.team, JSON.stringify({
```

Comment:
I think there is no need in 'scinote' prefix.
And I think we use '_' elsewhere in key names